docs/linter.md

```diff
@@ -369,6 +369,91 @@ Unlike isort, Ruff does not respect action comments within docstrings.
 See the [isort documentation](https://pycqa.github.io/isort/docs/configuration/action_comments.html)
 for more.
 
+## JSON output format
+
+The schema for JSON outputs is as follow:
+
+=== "Python"
+
+    ```python
+    type OneIndexed = int
```

Comment:
This might be a nit but I don't think this is correct because `OneIndexed` is a custom type defined at:

https://github.com/astral-sh/ruff/blob/f99af16bdae85829a0fa9b00168e7cad23bd7b00/crates/ruff_source_file/src/line_index.rs#L287-L294

which is a range from 1 to `usize::MAX` (depends on the platform). So, it wouldn't include 0 nor negative integers.

Reply:
How would you express this in Python -- via a subclass? `class OneIndexed(int): ...`?

Reply:
I think I'm more interested in understanding the use-case of this representation first. I agree that we should have the output format in some form in the documentation but I'm not sure what the format should be like.